tensorrt/0203-Fixed-samples-CMakeLists-for-conda-builds.patch

```diff
@@ -0,0 +1,25 @@
+From b5fad7d62ab294d70bf646323f622cf105090f3b Mon Sep 17 00:00:00 2001
```

Comment:
This patch fixes the main issue reported in https://github.com/open-ce/tensorrt-feedstock/issues/2

Reply:
No, it didn't work as this file just sets CUDA_INSTALL_DIR to /usr/local/cuda without checking if it is already set or not. This makes me think of another alternative that in this file, we add a check to see if it is already set. And if yes, then use that otherwise fallback to /usr/local/cuda.

Reply:
That would be a good upstream fix. If we need to patch the file anyways, we might as well just set it to the value that we need. So this current one should work.

Reply:
yeah, upstream here is actually not correct (or as correct as it could be). We could submit this upstream. Who knows if NVIDIA accepts things there though.